frontend/src/components/Body/Tarifs/Tariffs.js

```diff
@@ -33,6 +41,12 @@ const StyledButton = styled(Button)`
   }
 ` 
 
+const StyledButtonContainer = styled(Box)`
+  display: flex;
+  justify-content: center;
+  margin-top: auto;
+`;
+
```

Comment:
Remove this container and add a default "Box" instead. Move styles to a "sx" props also. Leave a comment near "Box" like "Container for styled button". Add same comments to other containers